paperqa/prompts.py

```diff
@@ -47,10 +47,11 @@
     "Papers: {papers}\n\n"
     "Selected keys:"
 )
+
 citation_prompt = (
     "Provide the citation for the following text in MLA Format. "
     "Do not write an introductory sentence. "
-    "If reporting date accessed, the current year is 2024\n\n"
+    "If reporting date accessed, the current year is 2025\n\n"
```

Comment:
```suggestion
    f"If reporting date accessed, the current year is {get_year()}\n\n"
```

And you'll need to import this